deploy/1.8+/metrics-server-deployment.yaml

```diff
@@ -33,6 +33,8 @@ spec:
         args:
           - --cert-dir=/tmp
           - --secure-port=4443
+          - --kubelet-insecure-tls
+          - --kubelet-preferred-address-types=InternalIP,ExternalIP,Hostname
```

Comment:
I would be ok with this change as MetricsServer should not depend on DNS (kube-dns can be configured to external).

Could you update other places that refer to this flag?
e.g. README in minikube section mentions adding this flag and e2e tests patch deployment https://github.com/kubernetes-sigs/metrics-server/blob/master/test/e2e.sh#L29